bundle/src/assembly/resources/fed/ns3/ns3_installer.sh

```diff
@@ -44,6 +44,7 @@ arg_ns3_file=""
 arg_federate_file=""
 arg_integration_testing=false
 arg_make_parallel=""
+arg_dev=false # set manually // do NOT remove src code after installation
```

Comment:
why don't you offer that as a cli-option, e.g. `--dev` ?

Reply:
`// do NOT remove src code after installation` sounds like a warning. Better rephrase to `// if set to true, source code is not removed after installation`